### PR TITLE
Updated Cassini driver and created tests

### PR DESCRIPTION
**Pull request by paarongiroux**
Added a model version property in the cassini driver. Changed the instrument_id to use the pds3label instrument_id rather than retrieving it from the label itself. Made pytests for the cassini drivers.

### CODE REVIEW

File: tests/pytests/test_cassini_drivers.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+data_naif.spice = simplespice
+cassini_drivers.spice = simplespice
+label_pds3.spice = simplespice
+
+from ale.drivers.cassini_drivers import CassiniIssPds3LabelNaifSpiceDriver
+
+CassiniIssPds3LabelNaifSpiceDriver.metakernel = get_mockkernels
+
+c = CassiniIssPds3LabelNaifSpiceDriver("")
```
Comment (jessemapel) [inline tests/pytests/test_cassini_drivers.py]: Pass this as a fixture to each test. Don't use a global variable.

File: tests/pytests/test_cassini_drivers.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import pytest
+
+import ale
+from ale.drivers import cassini_drivers
+from ale.base import data_naif
+from ale.base import label_pds3
+from ale.formatters import usgscsm_formatter
```
[the remark this answers is not in the document]
Comment (paarongiroux) [reply tests/pytests/test_cassini_drivers.py]: good spot, I removed it now.

File: tests/pytests/test_cassini_drivers.py (hunk 1 above)
Comment (paarongiroux) [reply tests/pytests/test_cassini_drivers.py]: I switched it to a fixture. I'm not too familiar with them so let me know if it looks alright.

**Comment by jessemapel** [comment]
Fixes #147

**Pull request merged**